src/main/target/SITL/target.c

```diff
@@ -79,6 +85,15 @@ int lockMainPID(void) {
 #define RAD2DEG (180.0 / M_PI)
 #define ACC_SCALE (256 / 9.80665)
 #define GYRO_SCALE (16.4)
+
+//#define SIMULATOR_IP "127.0.0.1"
+#define SIMULATOR_IP "172.21.32.1"
```

Comment:
Why we need a 172 private IP range here?

Reply:
I am still working on this. This IP is because I running SITL in WSL. I will eliminate this definition later.